packages/webpack/mixin.core.js

```diff
@@ -3,7 +3,7 @@ const { join } = require('path');
 
 const debug = require('debug')('untool:webpack:stats');
 const {
-  sync: { pipe, sequence },
+  sync: { pipe, sequence, override: overrideSync },
```

Comment:
Sorry if I was being unclear - I meant having both `overrideSync` and `overrideAsync`, as in..:

```javascript
const {
  sync: { sequence, pipe, override: overrideSync },
  async: { override: overrideAsync },
} = require('mixinable');
```